usr/src/cmd/cmd-inet/usr.bin/rcp.c

```diff
@@ -1827,7 +1827,7 @@ notzero(char *p, int n)
 {
 	register int result = 0;
 
-	while ((int)p & 3 && --n >= 0)
+	while ((intptr_t)p & 3 && --n >= 0)
```

Comment:
I think I'd just replace this entire function. It's just checking if a buffer is all zeros, but it's more complicated than it needs to be and uses dereferences which will be unaligned in 64-bit half the time.

Perhaps (not tested):
```c
int
is_empty(char *p, size_t n)
{
    return (p[0] == 0 && memcmp(p, p + 1, n - 1) == 0);
}
```

rcp is probably so legacy at this point that we could consider just not including it for new platforms at all.